### PR TITLE
[BD-26] Add prerequisites pages

### PR DESCRIPTION
**Pull request by viktorrusakov**
[EDUCATOR-5813](https://openedx.atlassian.net/browse/EDUCATOR-5813)

Added pages which handle prerequisites checks

### CODE REVIEW

File: src/instructions/proctored_exam/index.js [hunk 1]
```diff
@@ -4,5 +4,6 @@ export { default as SubmitProctoredExamInstructions } from './SubmitProctoredExa
 export { default as SubmittedProctoredExamInstructions } from './SubmittedProctoredExamInstructions';
 export { default as VerificationProctoredExamInstructions } from './VerificationProctoredExamInstructions';
 export { default as VerifiedProctoredExamInstructions } from './VerifiedProctoredExamInstructions';
-export { default as DownloadSoftwareProctoredExamInstructions } from './download-instructions';
+export { default as DownloadSoftwareProctoredExamInstructions } from './download-instructions/index';
```
Comment (ihor-romaniuk) [inline src/instructions/proctored_exam/index.js]: Is it necessary to have the `index` here? `index.js` should be as a default file in the path.

File: src/instructions/proctored_exam/index.js [hunk 2]
```diff
@@ -4,5 +4,6 @@ export { default as SubmitProctoredExamInstructions } from './SubmitProctoredExa
...
 export { default as ReadyToStartProctoredExamInstructions } from './ReadyToStartProctoredExamInstructions';
+export { default as PrerequisitesProctoredExamInstructions } from './prerequisites-instructions/index';
```
Comment (ihor-romaniuk) [inline src/instructions/proctored_exam/index.js]: and also here

File: src/instructions/proctored_exam/index.js (hunk 1 above)
Comment (viktorrusakov) [reply src/instructions/proctored_exam/index.js]: Yes, you're right, that's definitely not necessary, updated imports in new commit

Comment (ihor-romaniuk) [reply src/instructions/proctored_exam/index.js]: perfect!

**Review by ihor-romaniuk — APPROVED** [review]

**Review by UvgenGen — APPROVED** [review]

**Comment by idegtiarov** [comment]
Great, @ViktorRusakov please open PR into edx with the NOTE about PRs order

**Review by idegtiarov — APPROVED** [review]
Let's land it! ))

**Pull request merged**